pocs/tests/bisque/test_mount.py

```diff
@@ -1,6 +1,6 @@
 import os
-import pytest
 
+import pytest
```

Comment:
Is your editor doing this automatically?

Reply:
By this I assume you mean the reformatting of the order and spacing of import lines.